prometheus-metrics/src/main/scala/org/http4s/metrics/prometheus/Prometheus.scala

```diff
@@ -197,6 +197,8 @@ object Prometheus {
     * Given an exclude function, return a 'classifier' function, i.e. for application in
     * [[org.http4s.server/client.middleware.Metrics#apply]].
     *
+    * Note that the output String will be lower-cased.
```

Comment:
What are your thoughts on this, @ChristopherDavenport?

Reply:
Why do we want to lower-case it? Its case-insensitive, so its as sane a default as an in order to flatten them, but it does seem like an arbitrary choice.

Reply:
Yep - good point, Chris. I removed the `toLowerCase` in https://github.com/http4s/http4s/pull/3167#commits-pushed-badda65's commits.